.github/workflows/ci.yaml

```diff
@@ -100,17 +101,22 @@ jobs:
           containerfiles: |
            ./Dockerfile
           archs: amd64, arm64, ppc64le, s390x
-          tags: ${{ env.IMAGE}}:${{ needs.prepare.outputs.version }}
+          tags: ${{ env.IMAGE }}:${{ needs.prepare.outputs.version }}
           labels: ${{ needs.prepare.outputs.labels }}
           build-args: |
             GO_LINKER_ARGS=${{ needs.prepare.outputs.go_linker_args }}
           extra-args: |
-            --annotation=com.test.marketplace.product.service.name=services/dynatrace-bootsrapper-dynatrace-marketplace-test.something.boop
+            --annotation ${{ env.ANNOTATION}}
       - name: Push To quay.io
         id: push-to-quay
         uses: redhat-actions/push-to-registry@5ed88d269cf581ea9ef6dd6806d01562096bee9c # v2.8
         with:
           tags: ${{ steps.build-image.outputs.tags }}
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
+      - name: Add annotation to manifest
+        id: crane
```

Comment:
I guess you will need a step to login into the registry with crane if I am not mistaken

Reply:
this is what I am trying to test :)